### DISCUSSION TITLE
Download report ERR

### DISCUSSION THREAD

**Discussion by PaulRohlik** [frangoteam/FUXA#898] [Q&A] [answered]
Hello,
please help with downloading the report. Apparently I'm not authorized, but I'm logged into the editor as an admin. Can you advise what to do with it?

![fuxa pdf err](https://github.com/frangoteam/FUXA/assets/48689104/15854a9c-8594-482a-b2b1-a08f63df107f)

**Comment by unocelli** [comment, marked as answer]
Hi, Thanks for the report. Is a bag and we will fix it as soon as possible.

**Reply by PaulRohlik** [reply]
Hi,
very thanks :)
